doc/users-guide/base/configs.rst

```diff
@@ -0,0 +1,220 @@
+:title: Users Guide: Configuration Declarations
+
+.. _ug-typeAliases:
+
+**config** Declarations: command-line overrides
+===============================================
+
+The four main declaration keywords that have been introduced so far —
+``var``, ``const``, ``param``, and ``type`` — each support a mode in
+which their symbols' initial values as specified in the source code
+can be overridden on the command-line.  This supports the ability to
+specify and override default values, and is achieved by prefixing the
+statement with the keyword ``config``.
```

Comment:
The three comments on this paragraph are good.  I incorporated them and edited further, ending up with:

    The four main declaration keywords that have been introduced so far —
    ``var``, ``const``, ``param``, and ``type`` — support a mode in which
    their initializing expressions in the source code can be overridden on
    the command-line.  This	is achieved by prefixing the declaration with
    the keyword ``config``.  Chapel's ``config`` feature provides a simple
    means of specifying and	overriding default values.

Thoughts?